src/components/CurrentWeather/CurrentWeather.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import "./CurrentWeather.scss"
+// Importing Image
+import WeatherImages from '../../WeatherImages/WeatherImages'
+
+const CurrentWeather = ({weatherInfo}) => {
+
+  if(weatherInfo.length > 0){
+    
+  const infWeather =  weatherInfo[0].weather[0].id 
```

Comment:
As mentioned in my comment in `App.js`, this component should only receive `weatherInfo[0]` here, so then you will be able to extract what you need from that object directly in here